src/main/java/science/icebreaker/data/request/UpdateDeviceAvailabilityRequest.java

```diff
@@ -1,28 +1,31 @@
 package science.icebreaker.data.request;
 
-import javax.validation.constraints.NotBlank;
 import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
 
 public class UpdateDeviceAvailabilityRequest {
 
     private String comment;
     @Pattern(regexp = "[\\d]{5}", message = "Invalid postal code")
     private String germanPostalCode;
-    @NotBlank(message = "An institution name must be provided")
+    @Size(min = 1, message = "An institution name must be provided")
```

Comment:
Is it correct that null value is valid here?

Reply:
Yes. if the value is null, the older value is retained.